collector/textfile.go

```diff
@@ -65,6 +66,31 @@ func NewTextFileCollector() (Collector, error) {
 	}, nil
 }
 
+// Given a metric family, determine if any two entries are duplicates.
```

Comment:
Nitpick, docs haven't kept up with the code changes :) 
```suggestion
// Given a slice of metric families, determine if any two entries are duplicates.
```

Reply:
Good spot! I've fixed this in a recent push.